packages/shared/src/component-utils.ts

```diff
@@ -61,10 +61,14 @@ export const INTERNAL_doFormatting = (config?: ExpandedRouteConfig): string => {
   const key = allowedTypes[0];
   const formattedKey = formattedTypes[0];
 
-  const { maxFileSize, maxFileCount } = config[key]!;
+  const { maxFileSize, maxFileCount, minFileCount } = config[key]!;
 
   if (maxFileCount && maxFileCount > 1) {
-    return `${formattedKey}s up to ${maxFileSize}, max ${maxFileCount}`;
+    if (minFileCount > 0) {
```

Comment:
any reason default isn't 1?

Reply:
I had this as an open question in the pr description. Not sure which makes more sense

Reply:
no reason to send a request for 0 files - just an unnecessary roundtrip to the UT server